src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CodeActionEndpoint.cs

```diff
@@ -95,12 +95,6 @@ internal class CodeActionEndpoint : IVSCodeActionEndpoint
         public RegistrationExtensionResult GetRegistration(VSInternalClientCapabilities clientCapabilities)
         {
             _supportsCodeActionResolve = clientCapabilities.TextDocument?.CodeAction?.ResolveSupport != null;
-
-            if (!_languageServerFeatureOptions.RegisterBuiltInFeatures)
```

Comment:
We can remove the whole `RegisterBuiltInFeatures` property, right?

Reply:
Yep, opened up another PR just now to be explicit about it 🙂